app/views/trip_families/new.html.erb

```diff
@@ -0,0 +1,2 @@
+<p class="title"><%= t(".title") %></p>
```

Comment:
A suggestion to use `h1` wherever `<p class="title">` is used. Maybe a cleanup for another PR?

Reply:
👍